manifests/params.pp

```diff
@@ -38,6 +40,12 @@
         'libcurl4-openssl-dev',
         'perl-modules',
       ]
+
+      $convenience_packages = [
+        'screen',
+        'tree',
+        'vim',
```

Comment:
It would be nice to have emacs and possible tmux. I haven't used screen in at least 3 years.